src/components/RelatiesKoppelingen/RelatiesKoppelingen.js

```diff
@@ -66,21 +67,22 @@ const RelatiesKoppelingen = ({
     const [beleidsRelaties, setBeleidsRelaties] = React.useState([])
     const [isLoading, setIsLoading] = React.useState(true)
     const [activeTab, setActiveTab] = React.useState('Visueel')
-    const [verordeningsStructure, setVerordeningStructure] = React.useState(
-        null
-    )
+    const [verordeningsStructure, setVerordeningStructure] =
+        React.useState(null)
 
     // As the height of the containers will vary by the content, we make sure the user can immediately see the whole container by scrolling down
-    React.useEffect(() => window.scrollTo(0, document.body.scrollHeight), [
-        activeTab,
-    ])
+    React.useEffect(
```

Comment:
Could you check if you have a custom prettier config?

Reply:
Thanks to your prettier config settings, the code will not be automatically rewritten.